packages/editor/package.json

```diff
@@ -4,9 +4,9 @@
   "version": "1.0.0",
   "license": "GPL",
   "scripts": {
-    "start": "node scripts/start.js",
-    "build": "sh -ac '. ./.env; node scripts/build.js'",
-    "build:staging": "sh -ac '. ./.env.staging; node scripts/build.js'",
+    "start": "chmod +x ./env.sh && ./env.sh && node scripts/start.js",
+    "build": "sh -ac 'node scripts/build.js'",
+    "build:staging": "sh -ac 'node scripts/build.js'",
```

Comment:
I guess building for staging is not needed any longer as it is the same right?

Reply:
True that, removed